CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Release History
 
+> Release Date: future
+
+Enhancement:
+- Add "Set-SddcCertificateAuthority" cmdlet.
```

Comment:
```suggestion
- Add "Set-SddcCertificateAuthority" cmdlet. [GH-52](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-certificate-management/pull/52)
```

Reply:
addressed in (131acd4bec5c68a803eb55ef8c6b8d2638b81d17)